docs/spec/sources.md

```diff
@@ -362,5 +370,10 @@ _Structured Content Mode_ of the HTTP Protocol Binding for CloudEvents. Sources
 SHOULD send events to its
 [Destination](https://pkg.go.dev/github.com/knative/pkg/apis/duck/v1?tab=doc#Destination).
 
+A source that generates or forwards CloudEvents to its destination MUST produce
+or extend (respectively) the CloudEvents with the `knsource` extension, with
+value following the format `plural.group/name`. Off cluster sources MAY produce
+the `knsource` extension.
+
```

Comment:
Explain here what is difference between `knsource` and Cloud Event `source` and `subject` attribute? Somehing like:

```
The use case addressed is to identify sources that generate or forwards events in context of Knative cluster where eventing is installed. The existing Cloud Event attributes `source` and `subject` may be already present in forwarded event and changing them would be surprising to events consumers.
```

Reply:
Added, thanks!